### PR TITLE
L2-676: Simplify icp to tcycles conversion

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Simplify the code of the conversion of ICPs to TCycles and back.

# Changes

* Remove all calculations along the way.
* Utils gets the same exchange rate that CMC sends.
* Rename "ratio" to "exchangeRate".

# Tests

* Fix broken tests.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

File: frontend/svelte/src/lib/utils/icp.utils.ts [hunk 1]
```diff
@@ -74,21 +73,22 @@ export const convertNumberToICP = (amount: number): ICP => {
   return stake;
 };
 
+// `exchangeRate` is the number of 10,000ths of IMF SDR (currency code XDR) that corresponds to 1 ICP.
+// This value reflects the current market price of one ICP token.
+// https://sourcegraph.com/github.com/dfinity/ic@7cc18edf13bc7761bf58bf6e2eef558ba624adff/-/blob/rs/nns/cmc/cmc.did?L67
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/utils/icp.utils.ts:78]: can you update this link with the direct link to github?

File: frontend/svelte/src/lib/components/canisters/ConfirmCyclesCanister.svelte [hunk 2]
```diff
@@ -12,7 +12,10 @@
   let tCyclesFormatted: number | undefined;
   $: tCyclesFormatted =
     icpToCyclesRatio !== undefined
-      ? convertIcpToTCycles({ icpNumber: amount, ratio: icpToCyclesRatio })
+      ? convertIcpToTCycles({
+          icpNumber: amount,
+          exchangeRate: icpToCyclesRatio,
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/canisters/ConfirmCyclesCanister.svelte:17]: should we also rename the variables `icpToCyclesRatio` to `exchangeRate`?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

File: frontend/svelte/src/lib/utils/icp.utils.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/svelte/src/lib/utils/icp.utils.ts:78]: yes

File: frontend/svelte/src/lib/components/canisters/ConfirmCyclesCanister.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/canisters/ConfirmCyclesCanister.svelte:17]: agree

File: frontend/svelte/src/lib/utils/icp.utils.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/svelte/src/lib/utils/icp.utils.ts]: done

File: frontend/svelte/src/lib/components/canisters/ConfirmCyclesCanister.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/canisters/ConfirmCyclesCanister.svelte]: done

**Pull request merged**